install/upgrade/update_to_dev.php

```diff
@@ -39,7 +39,7 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->setVersion(PLUGIN_FLYVEMDM_VERSION);
 
    // update Entity config table
-   $table = PluginFlyvemdmEntityconfig::getTable();
+   $table = 'glpi_plugin_flyvemdm_entityconfigs';
```

Comment:
it is more robust to use raw table names here, if the itemtype is removed, the upgrade code still needs to handle it